app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -35,14 +35,15 @@ def custom_index_relation(collection)
 
     def_param_group :repo do
       param :url, String, :desc => N_("repository source url")
+      param :required_tags, Array, :desc => N_("Identifies whether the repository should be disabled on a client with a non-matching OS version. Pass [] to enable regardless of OS version. Maximum length 1; allowed tags are: #{Katello::RootRepository::ALLOWED_REQUIRED_TAGS.join(', ')}")
```

Comment:
I'd rather not call this parameter required_tags, which is candlepin-centric and not really matching up with how we identify it in the UI. Something like `os_releases` would be more appropriate imo

Reply:
per discussion, going with `os_versions`